src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,87 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public static final int defaultCapacity = 16;
```

Comment:
The constant 'defaultCapacity' should be named in uppercase as 'DEFAULT_CAPACITY' to follow Java naming conventions for constants.